app/controllers/patients_controller.rb

```diff
@@ -403,12 +405,11 @@ def update_history(patient, params, household, propagation)
     History.monitoring_status(history) if diff_state.include?(:monitoring)
```

Comment:
Could all of this be moved to a helper?

Reply:
I'm not thinking a Helper but rather move the logic to the History model. Maybe an entrypoint function that handles these types of history updates with propagation, from => to, etc. If it gets too much there then a module would be appropriate.

Reply:
ok that makes sense, thanks for the feedback Kyle!

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/371/commits/3b8d6282356126fff346b878902747a106eeb5d5